collections/_article/dev-snapshot-godot-4-4-dev-7.md

```diff
@@ -0,0 +1,175 @@
+took this idea one step further: integrating the tool as part of the engine *directly*. There was already a symbiosis
+between their team and the Godot engine, with many features being added to Godot *and* Jolt to accommodate both, but the
+integration of an official module was no small feat; their pull request
+([GH-99895](https://github.com/godotengine/godot/pull/99895)) ended up adding over 500 files and 115 **thousand** lines
+of code! So while this was one of the most rigorously tested PRs relative to the amount of time it's been up, it'd be
+impossible for any team to account for *everything* this behemoth introduced, so we eagerly await your feedback (and bug
+reports) on one of the most highly-requested features of 4.x.
+
+**NOTE:** At time of writing, this does **not** replace Godot Physics as the default 3D physics engine. This is because
+Jolt Physics has yet to reach full feature-parity with Godot Physics. If your interests/use-case are supported, the
+tool can be enabled by changing the `physics/3d/physics_engine` project setting to `Jolt Physics`.
+
+### .NET 8.0
+
+[Paul Joannon](https://github.com/paulloz) and [Raul Santos](https://github.com/raulsntos) have put the final pieces in
+place for supporting .NET 8 in Godot projects ([GH-92131](https://github.com/godotengine/godot/pull/92131) and
+[GH-100195](https://github.com/godotengine/godot/pull/100195)). All new projects will use .NET 8 by default and
+existing projects will automatically update to .NET 8 once opened with this release or any newer 4.4 build.
+
+### Improved Scene Tree editor performance
+
+The amount of performance boosts that have been crammed in between dev 6 and dev 7 has been staggering. Of note are the
+recent improvements to the engine's core including a many optimizes throughout the String class, spearheaded by
+[Ivorforce](https://github.com/Ivorforce). But perhaps the most obvious improvement comes from [HP van
+Braam](https://github.com/hpvb) and their modifications to the SceneTree system
+([GH-99700](https://github.com/godotengine/godot/pull/99700)). While this will prove beneficial across all projects,
+these changes result in especially complex scenes having a tenfold performance in the editor when moving or renaming
+nodes in the SceneTree.
+
+### Documentation tooltips
+
+Thanks to [Danil Alexeev](https://github.com/dalexeev) the GDScript code editor will now display a tooltip containing
+information about functions, variables, classes, etc. when you hover over them, including the documentation you have
+written using our documentation system ([GH-91060](https://github.com/godotengine/godot/pull/91060)). This makes using
+the integrated documentation system even more powerful as you no longer have to bounce between the code editor and the
+related docs to quickly get information.
+
+![Documentation tooltip](/storage/blog/dev-snapshot-godot-4-4-dev-7/documentation-tooltip.png)
```

Comment:
Should be converted to `.webp` too.

But maybe we can find an example that's a bit more interesting than a placeholder "Method description." on a dummy `method`? I'd show a tooltip for a known API method with documentation.

Reply:
Replaced with this as webp:
![image](https://github.com/user-attachments/assets/ec5d11bf-7cd9-4e9c-85d6-2148c4833cd8)